model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -342,6 +342,39 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
 
     # Nodes postprocessing stage in new graph
     for op in new_graph.get_op_nodes():
+        # tensor names restoring
+        debug_dict = {}
+
+        for out_port in op.ports:
+            out_tensor_names = op.ports[out_port][1]
+
+            # handle Constant operations with old style output port numbering
+            if op.soft_get('type') == 'Const':
+                assert len(op.ports) == 1, 'Something wrong with Constant node: {}, wrong number ' \
+                                           'of output ports: {}!'.format(op.soft_get('name'), len(op.ports))
+                if out_port == 1:
+                    out_port = 0
```

Comment:
`out_port = 0` instead of these two lines

Reply:
Done.